setup.py

```diff
@@ -17,7 +18,11 @@
     NO_EXTENSIONS = True
 
 
-if IS_GIT_REPO and not (HERE / "vendor/llhttp/README.md").exists():
+if (
+    not USE_SYSTEM_DEPS
```

Comment:
Technically, I think this block does not need to be executed with `NO_EXTENSIONS` but I didn't change the existing behavior. Please let me know if you prefer that I changed that.